src/customize-pop/keyboard-settings.md

```diff
@@ -0,0 +1,70 @@
+# Keyboard Settings
+
+Option to set input Sources, input source switching, special character entry options, and keyboard shortcuts are located in `Settings` ➞ `Keyboard`.
+
+![Keyboard Settings](/images/keyboard-settings/keyboard-settings.png)
+
+| Option | Function |
+|--------|----------|
+| Input Sources | Choose multiple input languages and layouts for a single keyboard. Sources can be moved up or down to determine priority. |
+| Input Source Switching | Set a keyboard shortcut for switching between your chosen sources. |
+| Special Character Key | Designate keys that enable entry of special characters. |
+| Keyboard Shortcuts | View, edit, and add keyboard shortcuts. |
+
+## Adding an Input Source
+
+1. Click the `+` to display available input sources. Select an input source from the list, or click the three vertical dots to perform a keyword search.
+
+    ![Display Sources](/images/keyboard-settings/display-sources.png)
+
+2. Select a language from the list.
+
+    ![Select Source](/images/keyboard-settings/select-source.png)
+
+3. Select a keyboard layout and click `Add`.
+
+> **Keyboard Layout vs. Keyboard Input Language**: A keyboard language refers to the available keys required to input the required characters and symbols for a given language. A keyboard layout is the physical organization of those keys.
+
+![Select Layout](/images/keyboard-settings/select-layout.png)
+
+Display keyboard layouts by clicking the three vertical dots on the right and clicking `View Keyboard Layout`
+
+![View Keyboard Layout](/images/keyboard-settings/view-keyboard-layout.png)
+
+Type while viewing the keyboard layout to verify accurate key registration.
+
+<video autoplay loop>
+    <source src="/images/keyboard-settings/view-layout.webm" />
+</video>
+<!--Link to info about keyboard configurator? https://system76.com/accessories/launch/download-->
+## Customizing Keyboard Shortcuts
+
+Pop!\_OS includes default shortcuts to provide intuitive and efficient navigation throughout the system. See [Navigate Pop!\_OS](/navigate-pop/navigate-pop.md) to learn about default window manipulation and navigation shortcuts in Pop!\_OS. Customize existing shortcuts and add new shortcuts in the `Keyboard Shortcuts` section.
+
+### Customize an Existing Shortcut
+
+1. Click `Customize Shortcuts` to view all keyboard shortcuts.
+
+    ![Customize Shortcuts](/images/keyboard-settings/customize-shortcuts.png)
+
+2. To add an additional key combination to an existing shortcut, click the three dots to the left and choose `Add another shortcut`.
```

Comment:
The three dots are to the right, not the left.

Your phrasing for the action ("existing shortcut") and the combination of keys required to activate the action ("an additional key combination") conflicts with the UI and screenshots' phrasing of "Add another shortcut." The UI seems to refer to the key combination itself as the shortcut, since those can be added to or removed from the available actions.

Reply:
Good point. I changed "shortcut" to "command" when referring to the result of the key combination.